### PR TITLE
Login component (add username to cookie)

### PR DESCRIPTION
**Pull request by JCTalbott**
and universal-cookie install

### CODE REVIEW

File: src/Login.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    return (
+        <Box display={showHideLogin} style={{ width: 400, margin: 'auto' }}>
+            <form>
+                <Form.Field>
+                    <Form.Label>Username</Form.Label>
+                    <Form.Control>
+                        <Form.Input
+                            color="success"
+                            value={username}
```
Comment (pdbradley) [inline src/Login.js:18]: for this default value for the input, maybe use placeholder?  This is from 

https://bulma.io/documentation/form/input/

image

Comment (pdbradley) [reply src/Login.js:18]: @JCTalbott @dinushkaherath

**Review by pdbradley — CHANGES_REQUESTED** [review]
@JCTalbott @dinushkaherath this is good to merge if you can change the default text of the input to use the placeholder attribute

File: src/Login.js (hunk 1 above)
Comment (dinushkaherath) [reply src/Login.js:19]: got it used placeholder instead of set state

**Pull request merged**